docker/README.md

```diff
@@ -0,0 +1,137 @@
+
+### Run a simtools-prod container
+
+To run the container in bash
+
+```bash
+docker run ghcr.io/gammasim/simtools-prod:latest bash
+```
+
+In the container, simtools applications are installed and can be called directly (e.g., `simtools-print-array-elements -h`).
+
+In case file exchange with the local file system is required, use the docker syntax to mount a directory. Example:
+
+```bash
+docker run --rm -it -v "$(pwd):/workdir/external" ghcr.io/gammasim/simtools-prod:latest bash
+```
+
+The following example runs an application inside the container and writes the output into a directory of the local files system,
+
+```bash
+docker run --rm -it -v "$(pwd):/workdir/external" \
+    ghcr.io/gammasim/simtools-prod:latest \
+    simtools-print-array-elements \
+    --array_element_list ./simtools/tests/resources/telescope_positions-North-utm.ecsv \
+    --export corsika --use_corsika_telescope_height \
+    --output_path /workdir/external/
+```
+
+Output files can be found `./simtools-output/`.
+
+### Building a simtools-prod container
+
+To build a new container locally run:
+
+```bash
+docker build -f Dockerfile-prod  -t simtools-prod .
+```
+
+Building will take a while and the image is large (~1.4 GB). For using images build on your own, replace in all examples `ghcr.io/gammasim/simtools-prod:latest` by the local image name `simtools-prod`.
+
+## Container for simtools Developers
+
+Provide a container for testing and development of simtools, including:
+
+- CORSIKA and sim\_telarray
+- packages required by simtools (from pyproject.toml)
+
+The container does not include the simtools code, which should be cloned in a separate directory (see below).
+
+### Run a container using the prepared Docker image available from repository
+
+Packages are available from the [simtools container repository](https://github.com/orgs/gammasim/packages?repo_name=simtools).
+
+**Preparation:**
+
+Create a new directory for your development and clone simtools into a subdirectory:
+
+```bash
+mkdir -p simtools-dev && simtools-dev
+git clone git@github.com:gammasim/simtools.git
+```
+
+To download and run a prepared container in bash:
+
+```bash
+docker run --rm -it -v "$(pwd)/:/workdir/external" ghcr.io/gammasim/simtools-dev:latest bash -c "$(cat ./simtools/docker/entrypoint.sh) && bash"
+```
+
+This additionally executes the `entrypoint.sh` script (e.g., for pip install or to set the database environment).
+
+Remember you need to `docker login` to the GitHub package repository with a personal token in order to download an image (follow [these instructions](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)).
+
+### Build a new developers container locally
+
+To build a new container locally run:
+
+```bash
+docker build -f Dockerfile-dev -t simtools-dev .
+```
+
+Use the docker container in the same way as above, replacing `ghcr.io/gammasim/simtools-dev:latest` by `simtools-dev`.
```

Comment:
not a big issue but it is better to quote the entire command instead. As I commented above, please include the path to the correct repository before this command, i.e., since we are at this point inside the `simtools/docker` repository:

```bash
cd ../..
docker run --rm -it -v "$(pwd)/:/workdir/external" simtools-dev bash -c "$(cat ./simtools/docker/entrypoint.sh) && bash"

Reply:
But this would be a repetition? I think the instructions are clear and repeating means that if this changes in future, we have one more line to correct (or miss it). I would prefer to leave it as it is.

Reply:
Ok, changing in only one place in the future is a good argument.